Add support for FPU flag for PDs (#404)
seL4/microkit: docs/manual.md

  Must be be between 4KiB and 16MiB and be 4K page-aligned. Defaults to 8KiB.
* `cpu`: (optional) set the physical CPU core this PD will run on. Defaults to zero.
* `smc`: (optional, only on ARM) Allow the PD to give an SMC call for the kernel to perform.. Defaults to false.

> * `fpu`: (optional) disable the FPU support for this PD. Defaults to true.

Reviewer: ```suggestion
* `fpu`: (optional) whether this PD can access the FPU. Defaults to true.
```

  Must be be between 4KiB and 16MiB and be 4K page-aligned. Defaults to 8KiB.
* `cpu`: (optional) set the physical CPU core this PD will run on. Defaults to zero.
* `smc`: (optional, only on ARM) Allow the PD to give an SMC call for the kernel to perform.. Defaults to false.

> * `fpu_disabled`: (optional) whether this PD can access the FPU. Defaults to false.

Reviewer: Just because seL4 has it a particular way doesn't mean the Microkit level has to specify it the same way. Since Microkit gets to choose its own defaults it can just leave it as defaulting to true.

Reviewer: I will revert the commit that changes it to `fpu_disabled`, having just `fpu` is much easier to read.

Author: Okay, I thought we are aiming for consistency - I think it saves developers head-scratching if we keep everything consistent. But I am not opposed.

Author: As discussed offline - it makes more sense to stick to the `fpu` convention instead. Closing.
